### ISSUE TITLE
Support Swift 4.2

### ISSUE THREAD

**Issue by jordanekay** [JohnSundell/Marathon#178]
I would love to use Marathon run scripts written in Swift 4.2. Thanks!

**Comment by JohnSundell** [comment]
Me too :) https://github.com/JohnSundell/Marathon/pull/181

**Comment by CalQL8ed-K-OS** [comment]
I think lack of Swift 4.2 support is the reason I'm getting this output, ¿thoughts @JohnSundell?
```
$ marathon install johnsundell/playground
🏃  Cloning https://github.com/johnsundell/playground.git...
   Compiling script...
💥  Failed to compile script
👉  The following error(s) occured:
   - 126:33: value of type '[String]' has no member 'compactMap'
```

**Comment by JohnSundell** [comment]
@CalQL8ed-K-OS `compactMap` was introduced in Swift 4.1. Are you sure you're running 4.1?

**Comment by JohnSundell** [comment]
Fixed in #181
